### PR TITLE
Updates toCsTenantDialPlan and Set-CsUser

### PR DESCRIPTION
**Pull request by JensTrier**
Updating *-CsTenantDialPlan and adding a deprecation note to Set-CsUser for -HostedVoicemail parameter

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by get-itips — APPROVED** [review]

File: skype/skype-ps/skype/Set-CsTenantDialPlan.md [hunk 1]
```diff
@@ -67,12 +54,11 @@ $NR.Name = "RedmondRule"
...
-We then assign the string RedmondRule to the Name property of the object.
-Finally, we pass the variable back to the NormalizationRules parameter of the `Set-CsTenantDialPlan` cmdlet to make the change permanent.
-
+This example changes the name of an normalization rule. Keep in mind that changing the name also changes the name portion of the Identity.
+The `Set-CsVoiceNormalizationRule` cmdlet doesn't have a Name parameter, so in order to change the name, we first call the `Get-CsTenantDialPlan` cmdlet to retrieve the Dial Plan with
+the Identity Global and assign the returned object to the variable $DP. Then we filter the NormalizationRules Object for the rule RedmondFourDigit and assign the returned object to
+the variable $NR. We then assign the string RedmondRule to the Name property of the object. Finally, we pass the variable back to the NormalizationRules parameter of the
+`Set-CsTenantDialPlan` cmdlet to make the change permanent.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTenantDialPlan.md:61]:
```suggestion
This example changes the name of a normalization rule. Keep in mind that changing the name also changes the name portion of the Identity.
The `Set-CsVoiceNormalizationRule` cmdlet doesn't have a Name parameter, so in order to change the name, we first call the `Get-CsTenantDialPlan` cmdlet to retrieve the Dial Plan with
the Identity Global and assign the returned object to the variable $DP. Then we filter the NormalizationRules Object for the rule RedmondFourDigit and assign the returned object to
the variable $NR. We then assign the string RedmondRule to the Name property of the object. Finally, we pass the variable back to the NormalizationRules parameter of the
`Set-CsTenantDialPlan` cmdlet to make the change permanent.
```

File: skype/skype-ps/skype/Set-CsTenantDialPlan.md [hunk 2]
```diff
@@ -67,12 +54,11 @@ $NR.Name = "RedmondRule"
 Set-CsTenantDialPlan -Identity Global -NormalizationRules $DP.NormalizationRules
 ```
 
-This example changes the name of an normalization rule.
-Keep in mind that changing the name also changes the name portion of the Identity.
-The `Set-CsVoiceNormalizationRule` cmdlet doesn't have a Name parameter, so in order to change the name, we first call the `Get-CsTenantDialPlan` cmdlet to retrieve the Dial Plan with the Identity Global and assign the returned object to the variable $DP. Then we filter the NormalizationRules Object for the rule RedmondFourDigit and assign the returned object to the variable $NR.
-We then assign the string RedmondRule to the Name property of the object.
-Finally, we pass the variable back to the NormalizationRules parameter of the `Set-CsTenantDialPlan` cmdlet to make the change permanent.
-
+This example changes the name of an normalization rule. Keep in mind that changing the name also changes the name portion of the Identity.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTenantDialPlan.md:57]:
```suggestion
This example changes the name of a normalization rule. Keep in mind that changing the name also changes the name portion of the Identity.
```

File: skype/skype-ps/skype/Set-CsTenantDialPlan.md [hunk 3]
```diff
@@ -231,35 +181,15 @@ Accept wildcard characters: False
 ```
 
 ### -SimpleName
-The SimpleName parameter is a display name for the tenant dial plan.
-This name must be unique among all tenant dial plans.
+The SimpleName parameter is a display name for the tenant dial plan. This name must be unique among all tenant dial plans.
 
-This string can be up to 49 characters long.
-Valid characters are alphabetic or numeric characters, hyphen (-), dot (.) and parentheses (()).
+This string can be up to 49 characters long. Valid characters are alphabetic or numeric characters, hyphen (-), dot (.) and parentheses (()).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTenantDialPlan.md:186]:
```suggestion
This string can be up to 49 characters long. Valid characters are alphabetic or numeric characters, hyphen (-), dot (.), and parentheses (()).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the two additional changes. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the two additional changes. Many thanks.

Thanks & Done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**